Changelog.md

```diff
@@ -1,4 +1,5 @@
 ## Unreleased
+* Support SCRAM authentication, use plain passwords in auth_file
```

Comment:
(optional) Would it be worth mentioning the change in `server_tls_sslmode` in the changelog here (as a separate bullet) too?